WordPress/UITests/Tests/DashboardTests.swift

```diff
@@ -3,6 +3,8 @@ import XCTest
 
 // These tests are Jetpack only.
 class DashboardTests: XCTestCase {
+
+    @MainActor
```

Comment:
All this means is that the method will run on the main thread. There reason all the tests have been updated is explained later on in the `setUpTestSuite()` diff.